frontend/src/Components/Dashboards/DashboardVolunteers.jsx

```diff
@@ -5,28 +5,164 @@ DashboardVolunteers Component | Capstone App (Pursuit Volunteer Mgr)
 
 
 /* IMPORTS */
-import React from 'react'
 
-const DashboardVolunteers = () => {
-  return (
-    <div className="dashboard mt-4">
-      
-      <h2>Volunteer's Dashboard</h2><br />
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
-      <strong>Events</strong><br />
-      Upcoming events<br />
-      Newly added events<br />
-      Events signed up for<br /><br />
+// import VolunteerPreviewCard from './VolunteerPreviewCard';
+import EventPreviewCard from '../EventPreviewCard';
+import EventsCard from '../EventCard';
 
-      <strong>Mentoring</strong><br />
-      Mentees?<br /><br />
+const Dashboard = (props) => {
+    const { setFeedback, loggedUser } = props;
+    const [eventsList, setEventsList] = useState([]);
+    const [showEvent, setShowEvent] = useState(false);
+    const [showVolunteeredTime, setShowVolunteeredTime] = useState(0);
+    const [showPastEvents, setShowPastEvents] = useState('')
+    const [showImportantEvents, setShowImportantEvents] = useState([]);
+    const [targetEvent, setTargetEvent] = useState({});
+    const [reload, setReload] = useState(false);
 
-      <strong>Some My Profile Data</strong><br />
-      Banked Volunteer hours (by last 7 days? total? banked?)<br />
-      Link to edit profile<br /><br />
 
-    </div>
-  );
+    useEffect(() => {
+        const getEvents = async () => {
+            try {
+                const { data } = await axios.get(`/api/events/upcoming/volunteer/${props.loggedUser.v_id}`);
```

Comment:
I didn't use the limit on the query because I was unsure if we wanted the admin to have access to more than just 3 important events on their dashboard.

Reply:
I added the limit tot the query